### PR TITLE
Top level pages structure

### PR DESCRIPTION
**Pull request by paulotruta**
This PR adds top level pages and their basic content.
This is not intended to be a full point-to-point update to the pages themselves, but just to setup the base structure of the site, and add some tweaks along the way.

### CODE REVIEW

**Comment by paulotruta** [comment]
Added a couple of pages, with special highlight to the code of conduct and contribution page. Will continue adding some more basic info (installation, configuration, execution) in the next day or two.

File: src/contributing.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Contributing
+
+Thank you for your interest in making a contribution! 
+
+All changes, small or big, are welcome, and regardless of who you are, we are always happy to work together to make your contribution as strong as possible. We follow the [Covenant Code of Conduct](../code_of_conduct), so we ask you to uphold it. Be kind to everyone in the community, and please report unacceptable behavior to info@edgebox.io
```
Comment (inverse) [inline src/contributing.md]:
```suggestion
All changes, small or big, are welcome, and regardless of who you are, we are always happy to work together to make your contribution as strong as possible. We follow the [Covenant Code of Conduct](/others/code-of-conduct), so we ask you to uphold it. Be kind to everyone in the community, and please report unacceptable behavior to info@edgebox.io
```

URL was 404

**Review by inverse — APPROVED** [review]
Looks good aside the comment - awesome improvements!

File: src/contributing.md (hunk 1 above)
Comment (paulotruta) [reply src/contributing.md]: Nice, good catch! 💯

**Pull request merged**